server/container_create.go

```diff
@@ -384,6 +384,27 @@ func ensureSaneLogPath(logPath string) error {
 	return nil
 }
 
+// addSecretsBindMounts mounts user defined secrets to the container
+func addSecretsBindMounts(mountLabel, ctrRunDir, secretsFilePath string, specgen generate.Generator) error {
+	// secretsFilePath is used to override the mount file path for testing purposes only
+	mountPaths := [3]string{secretsFilePath, libkpod.OverrideSecretsMountFile, libkpod.DefaultSecretsMountFile}
+	for _, path := range mountPaths {
+		containerMounts := specgen.Spec().Mounts
+		mounts, err := secretMounts(mountLabel, path, ctrRunDir, containerMounts)
+		if err != nil {
+			return err
+		}
+		for _, m := range mounts {
+			specgen.AddBindMount(m.Source, m.Destination, nil)
+
+		}
+		if secretsFilePath != "" {
```

Comment:
ugh, can you:
```go
mountPaths := []string{libkpod.OverrideSecretsMountFile, libkpod.DefaultSecretsMountFile}
if secretsFilePath != "" {
    mountPaths = append([]string{secretsFilePath}, mountPaths...)
}
```
and remove this break here?

Reply:
sure, was just trying to avoid repeating code.

Reply:
fixed